app/updates/StructuredLogger.scala

```diff
@@ -9,7 +9,7 @@ import services.ConfigAgent
 import scala.collection.JavaConverters._
 
 class StructuredLogger(val config: ApplicationConfiguration, val configAgent: ConfigAgent) extends Logging {
-  def putLog(log: LogUpdate, level: String = "info", error: Option[Exception] = None): Unit = {
+  def putLog(log: LogUpdate, level: String = "info", error: Option[Throwable] = None): Unit = {
```

Comment:
Are you sure you want `Throwable` ?

It's usually reserved for when you want to catch `Exception`s and `Error`s such as`OutOfMemoryError`

Reply:
Agreed. I've dropped it back to `Exception`